README.md

```diff
@@ -0,0 +1,14 @@
+# IcedTeaWeb
+This repository contains all sources of IcedTeaWeb. 
+
+## About IcedTeaWeb
+IcedTeaWeb is an open source implementation of JSR-56 that is better know as WebStart.
```

Comment:
I would add "Java" the the end of the sentence to make description compliant to Wikipedia: "...that is better know as Java Web Start."

Reply:
can you do this?

Reply:
yes, will make my changes on the branch